modules/smtp/modules.php

```diff
@@ -104,6 +104,53 @@ public function process() {
     }
 }
 
+/**
+ * @subpackage smtp/handler
+ */
+class Hm_Handler_load_smtp_is_imap_forward extends Hm_Handler_Module
+{
+    public function process() {
+        if (!$this->module_is_supported('imap')) {
+            Hm_Msgs::add('ERRIMAP module unavailable.');
```

Comment:
You could forward a POP3 message so while I don't care about attachments in that case we probably don't need this user facing warning, just the early exit.

Reply:
I remove the warning message, @jasonmunro can you review please?